### PR TITLE
1.12.2 h77cc08

### PR DESCRIPTION
**Pull request by chaen**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by chrisburr** [comment]
@conda-forge-admin, please restart ci

**Comment by chrisburr** [comment]
This is failing because of https://github.com/abdes/cryptopp-cmake/pull/113

**Comment by chrisburr** [comment]
@conda-forge-admin, please restart ci

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,7 +31,7 @@ requirements:
   host:
     - python
     - pip
-    - gfal2
+    - gfal2 2.22.1
```
Comment (chaen) [inline recipe/meta.yaml:34]: Is pinning the version necessary here ?

Comment (chrisburr) [reply recipe/meta.yaml:34]: I messed up, I forgot that `gfal2` is in the global conda-forge pinning so the migration needed to be triggered.

(it doesn't really matter, but this was working around the global pinning)

**Pull request merged**